digitalocean/resource_digitalocean_image.go

```diff
@@ -0,0 +1,251 @@
+package digitalocean
+
+import (
+	"context"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/digitalocean/godo"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+// Ref: https://developers.digitalocean.com/documentation/v2/#retrieve-an-existing-image-by-id
+const (
+	imageAvailableStatus = "available"
+	imageDeletedStatus   = "deleted"
+)
+
+func resourceDigitalOceanImage() *schema.Resource {
```

Comment:
```suggestion
func resourceDigitalOceanCustomImage() *schema.Resource {
```

The methods names as well.

Reply:
resolved